### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/response-model.md`

### PR DESCRIPTION
**Pull request by AndreBBM**

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

File: docs/pt/docs/tutorial/response-model.md [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+* `@app.post()`
+* `@app.put()`
+* `@app.delete()`
+* etc.
+
+{* ../../docs_src/response_model/tutorial001_py310.py hl[17,22,24:27] *}
+
+/// note
+
+Observe que `response_model` é um parâmetro do método "decorator" (`get`, `post`, etc). Não da sua *função de operação de caminho (path operation function)*, como todos os parâmetros e corpo.
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:44]: You can change

*função de operação de caminho (path operation function)*

to

*função de operação de rota*

File: docs/pt/docs/tutorial/response-model.md [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+
+///
+
+`response_model` recebe o mesmo tipo que você declararia para um campo de modelo Pydantic, então, pode ser um modelo Pydantic, mas também pode ser, por exemplo, uma `lista` de modelos Pydantic, como `List[Item]`.
+
+O FastAPI usará este `response_model` para fazer toda a documentação de dados, validação, etc. e também para **converter e filtrar os dados de saída** para sua declaração de tipo.
+
+/// tip | dica
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:52]: you can start dica with a capital letter: 

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/response-model.md [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+
+Se você tiver verificações de tipo rigorosas em seu editor, mypy, etc, você pode declarar o tipo de retorno da função como `Any`.
+
+Dessa forma, você diz ao editor que está retornando qualquer coisa intencionalmente. Mas o FastAPI ainda fará a documentação de dados, validação, filtragem, etc. com o `response_model`.
+
+///
+
+### Prioridade `response_model`
+
+Se você declarar ambos um tipo de retorno e um `response_model`, o `response_model` terá prioridade e será usado pelo FastAPI.
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:62]: It got a little strange here with "ambos um tipo de retorno..."

I would suggest something like:

"Se você declarar tanto um tipo de retorno quanto um response_model, o response_model terá prioridade e será usado pelo FastAPI."

File: docs/pt/docs/tutorial/response-model.md [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+
+Dessa forma, você pode adicionar anotações de tipo corretas às suas funções, mesmo quando estiver retornando um tipo diferente do modelo de resposta, para ser usado pelo editor e ferramentas como mypy. E ainda assim você pode fazer com que o FastAPI faça a validação de dados, documentação, etc. usando o `response_model`.
+
+Você também pode usar `response_model=None` para desabilitar a criação de um modelo de resposta para essa *operação de caminho (path operation)*, você pode precisar fazer isso se estiver adicionando anotações de tipo para coisas que não são campos Pydantic válidos, você verá um exemplo disso em uma das seções abaixo.
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:66]: You can replace *operação de caminho (path operation)* with *operação de rota*

File: docs/pt/docs/tutorial/response-model.md [hunk 5]
```diff
@@ -0,0 +1,357 @@
...
+
+## Retorna os mesmos dados de entrada
+
+Aqui estamos declarando um modelo `UserIn`, ele conterá uma senha em texto simples:
+
+{* ../../docs_src/response_model/tutorial002_py310.py hl[7,9] *}
+
+/// info
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:74]: You can add the translation to this admonition:

```
/// info | Informação
```

File: docs/pt/docs/tutorial/response-model.md [hunk 6]
```diff
@@ -0,0 +1,357 @@
...
+
+E estamos usando este modelo para declarar nossa entrada e o mesmo modelo para declarar nossa saída:
+
+{* ../../docs_src/response_model/tutorial002_py310.py hl[16] *}
+
+Agora, sempre que um navegador estiver criando um usuário com uma senha, a API retornará a mesma senha na resposta.
+
+Neste caso, pode não ser um problema, porque é o mesmo usuário enviando a senha.
+
+Mas se usarmos o mesmo modelo para outra *operação de caminho (path operation)*, poderíamos estar enviando as senhas dos nossos usuários para todos os clientes.
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:100]: You can use *operação de rota*

File: docs/pt/docs/tutorial/response-model.md [hunk 7]
```diff
@@ -0,0 +1,357 @@
...
+
+/// danger | perigo
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:102]: You can use a capital letter:

```
/// danger | Perigo
```

But this danger is more about something to pay attention to.... There are translations to danger in portuguese docs that translates it to "Cuidado". I think it's more appropriate

File: docs/pt/docs/tutorial/response-model.md [hunk 8]
```diff
@@ -0,0 +1,357 @@
...
+
+///
+
+## Adicionar um modelo de saída
+
+Podemos, em vez disso, criar um modelo de entrada com a senha em texto simples e um modelo de saída sem ela:
+
+{* ../../docs_src/response_model/tutorial003_py310.py hl[9,11,16] *}
+
+Aqui, embora nossa *função de operação de caminho (path operation function)* esteja retornando o mesmo usuário de entrada que contém a senha:
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:114]: You can use *função de operação de rota*

File: docs/pt/docs/tutorial/response-model.md [hunk 9]
```diff
@@ -0,0 +1,357 @@
...
+
+///
+
+/// info
+
+O FastAPI usa `.dict()` do modelo Pydantic com <a href="https://docs.pydantic.dev/1.10/usage/exporting_models/#modeldict" class="external-link" target="_blank">seu parâmetro `exclude_unset`</a> para chegar a isso.
+
+///
+
+/// info
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:267]: And here:

```
/// info | Informação
```

File: docs/pt/docs/tutorial/response-model.md [hunk 10]
```diff
@@ -0,0 +1,357 @@
...
+"tax": 10.5,
+"tags": []
+}
+```
+
+O FastAPI é inteligente o suficiente (na verdade, o Pydantic é inteligente o suficiente) para perceber que, embora `description`, `tax` e `tags` tenham os mesmos valores que os padrões, eles foram definidos explicitamente (em vez de retirados dos padrões).
+
+Portanto, eles serão incluídos na resposta JSON.
+
+/// tip | dica
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:311]: You can use a capital letter here:

```
/// tip | Dica
```

File: docs/pt/docs/tutorial/response-model.md [hunk 11]
```diff
@@ -0,0 +1,357 @@
...
+
+Observe que os valores padrão podem ser qualquer coisa, não apenas `None`.
+
+Eles podem ser uma lista (`[]`), um `float` de `10.5`, etc.
+
+///
+
+### `response_model_include` e `response_model_exclude`
+
+Você também pode usar os parâmetros `response_model_include` e `response_model_exclude` do *path operation decorator*.
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:321]: You can translate to *decorador de operação de rota*

File: docs/pt/docs/tutorial/response-model.md [hunk 12]
```diff
@@ -0,0 +1,357 @@
...
+
+Eles pegam um `set` de `str` com o nome dos atributos para incluir (omitindo o resto) ou para excluir (incluindo o resto).
+
+Isso pode ser usado como um atalho rápido se você tiver apenas um modelo Pydantic e quiser remover alguns dados da saída.
+
+/// tip | dica
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:327]: You can use a capital letter here:

/// tip | Dica

File: docs/pt/docs/tutorial/response-model.md [hunk 13]
```diff
@@ -0,0 +1,357 @@
...
+
+Isso ocorre porque o Schema JSON gerado no OpenAPI do seu aplicativo (e a documentação) ainda será o único para o modelo completo, mesmo que você use `response_model_include` ou `response_model_exclude` para omitir alguns atributos.
+
+Isso também se aplica ao `response_model_by_alias` que funciona de forma semelhante.
+
+///
+
+{* ../../docs_src/response_model/tutorial005_py310.py hl[29,35] *}
+
+/// tip | dica
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:339]: You can use a capital letter here:

/// tip | Dica

File: docs/pt/docs/tutorial/response-model.md [hunk 14]
```diff
@@ -0,0 +1,357 @@
...
+
+A sintaxe `{"nome", "descrição"}` cria um `conjunto` com esses dois valores.
+
+É equivalente a `set(["nome", "descrição"])`.
+
+///
+
+#### Usando `listas` em vez de `set`s
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:347]: I think you can remove the translation to lists here:

```
Usando `list`s em vez de `set`s
```

File: docs/pt/docs/tutorial/response-model.md [hunk 15]
```diff
@@ -0,0 +1,357 @@
...
+
+Se você esquecer de usar um `set` e usar uma `lista` ou `tupla` em vez disso, o FastAPI ainda o converterá em um `set` e funcionará corretamente:
+
+{* ../../docs_src/response_model/tutorial006_py310.py hl[29,35] *}
+
+## Recapitulação
+
+Use o parâmetro `response_model` do *path operation decorator* para definir modelos de resposta e, especialmente, para garantir que dados privados sejam filtrados.
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/response-model.md:355]: You can translate to *decorador de operação de rota*.

**Review by ceb10n — APPROVED** [review]
Really great work @AndreBBM 

I didn't find anything wrong. But I've added some suggestions to keep the standards for the portuguse translation...

It's approved since there are only suggestions, but feel free to add them if you want :D

**Review by ceb10n — APPROVED** [review]
🚀

**Review by nazarepiedady — APPROVED** [review]
Thanks for your collaboration, @AndreBBM.

**Comment by alejsdev** [comment]
Thanks for your contribution! @AndreBBM :100:  
And thanks for your review @ceb10n :mag:

**Pull request merged**